commands/mta_command.go

```diff
@@ -120,26 +147,56 @@ func (c *MtaCommand) executeInternal(positionalArgs []string, dsHost string, fla
 }
 
 func size(n int64) string {
-	return formatters.ByteSize(n * formatters.MEGABYTE)
+	return formatters.ByteSize(n)
 }
 
-func getInstances(app plugin_models.GetAppsModel) string {
-	return strconv.Itoa(app.RunningInstances) + "/" + strconv.Itoa(app.TotalInstances)
+func getInstances(processes []models.ApplicationProcessStatistics) string {
+	var runningProcesses []models.ApplicationProcessStatistics
+	for _, process := range processes {
+		if process.State == "RUNNING" {
+			runningProcesses = append(runningProcesses, process)
+		}
+	}
+	return strconv.Itoa(len(runningProcesses)) + "/" + strconv.Itoa(len(processes))
 }
 
-func getRoutes(app plugin_models.GetAppsModel) string {
+func sumMemory(processes []models.ApplicationProcessStatistics) int64 {
+	var result int64
+	for _, process := range processes {
+		result += process.Memory
+	}
+	return result
```

Comment:
for 5 processes won't the sum be: 5 * (4 for example) = 20GB, shouldn't we print just 4GB and the number of instances?

Reply:
Shouldn't the `memory` field of the table specify _total_ memory? If an app has 5 instances it will consume 20GB, not 4.

Reply:
Yes, it will use 20, but not it shows, the memory of one instance:

```
Apps:
name     requested state   instances   memory   disk   urls
test12   started           3/3         1G       1G     deploy-service-i500571-test12.cfapps.sap.hana.ondemand.com
```
The cf cli itself also shows the memory of one instance:
`
type:           web
sidecars:
instances:      3/3
memory usage:   1024M
     state     since                  cpu    memory        disk           details
#0   running   2022-07-15T16:14:55Z   1.1%   69.9M of 1G   123.1M of 1G
#1   running   2022-07-18T07:48:49Z   1.0%   66.7M of 1G   123.1M of 1G
#2   running   2022-07-18T07:48:50Z   3.4%   66.3M of 1G   123.1M of 1G
`

Reply:
Changed to only display memory and disk of 1 process